### ISSUE TITLE
Clear up error message when production build is missing during `next start`

### ISSUE THREAD

**Issue by timneutkens** [vercel/next.js#19778]
From @jlongster

**yarn start didn't work**

Steps:

```jsx
$ git clone https://github.com/colinhacks/devii.git
$ yarn
$ yarn start
```

Error:

```jsx
Error: Could not find a valid build in the '<redacted>/.next' directory! Try building your app with 'next build' before starting the server.
    <redacted>
error Command failed with exit code 1.
```

`yarn start` is usually the way you start developing. Looking at the readme of the repo I saw it says to use `yarn dev`. Doing that works, but looking at `package.json` I see scripts that map to `next dev` and `next start`. Don't know why there are two different commands that are similar.

The above error message should be more helpful

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
